model/Security/Classes/VexNotAffectedVulnAssessmentRelationship.md

```diff
@@ -4,25 +4,49 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+Links a vulnerability and one or more elements designating the latter as products
+not affected by the vulnerability.
 
 ## Description
 
-An VexNotVulnAffectedAssessment is TODO
+`VexNotVulnAffectedAssessmentRelationship` connects a vulnerability and a number
```

Comment:
this code uses backtics for formatting code which is a markdown feature which might not be supported by the parser and the documentation generation (yet). These would just be added verbatim to the documentation and probably should not be used.